test/test_cmd.py

```diff
@@ -109,3 +116,42 @@ def test_file_types(self):
         self.assertEqual(0, self.call_command(["write", JUST_TESTING]))
         self.assert_file(JUST_TESTING, contains="\n-- ** DO NOT EDIT")
         self.assert_file(JUST_TESTING, contains="you must edit the central file", not_contains="LOCAL CHANGE")
+
+
+class UpdateCommandTest(CommandTest):
+    """Tests for the update command."""
+
+    def test_no_files_exist(self):
+        self.assertEqual(0, self.call_command(["update"]))
+        self.assert_not_file(PYLINTRC)
+        self.assert_not_file(EDITORCONFIG)
+
+    def test_one_file_exists(self):
+        # Write the initial version of pylintrc.
+        self.assertEqual(0, self.call_command(["write", PYLINTRC]))
+        self.assert_file(PYLINTRC, contains="[MASTER]")
+        self.assert_not_file(EDITORCONFIG)
+
+        # Pretend edx-lint has a newer version of pylintrc, and update files.
+        with patch("edx_lint.write.get_file_content") as get_file_content:
+            get_file_content.return_value = "[xyz]\nsetting = True"
+            self.assertEqual(0, self.call_command(["update"]))
+
+        # pylintrc is re-written with new content.
+        self.assert_file(PYLINTRC, contains="setting = True")
+        self.assert_file(PYLINTRC, not_contains="[MASTER]")
+        # .editorconfig still hasn't been written.
+        self.assert_not_file(EDITORCONFIG)
+
+    def test_update_wont_clobber(self):
+        # Create a hand-written pylintrc file:
+        with open(PYLINTRC, "w") as pylintrc:
+            pylintrc.write("[bogus]\nnot_really = pylintrc\n")
+
+        # Update will see it's not one made by edx-lint, and move it aside.
+        self.assertEqual(0, self.call_command(["update"]))
+
+        # The backup file is our original hand-written file.
+        self.assert_file(PYLINTRC_BACKUP, contains="not_really = pylintrc")
```

Comment:
Should this be a check in the original file if we want to test that it wasn't clobbered?

Reply:
edx-lint checks the existing file.  If it wasn't written by edx-lint, then it moves the existing file to a backup, and then writes the new file.  So when write (or update) is done, the real file is always the recent edx-lint output.  "Not clobbering" the hand-written file might be a misnomer.  The data hasn't been lost, but it has been shoved aside.

Reply:
Got it so it's more like `test_update_will_backup`.  Please add your reply or something like it as a comment in this test function.